scripts/st2bootstrap-el8.template.sh

```diff
@@ -163,8 +168,10 @@ install_st2_dependencies() {
   sudo systemctl start rabbitmq-server
   sudo systemctl enable rabbitmq-server
 
-  # Various other dependencies needed by st2 and installer script
-  sudo yum -y install crudini
+  sudo rabbitmqctl add_user stanley "${ST2_RABBITMQ_PASSWORD}"
```

Comment:
Can you change the username from `stanley` to `stackstorm`?  This is to be consistent with the username that we used for MongoDB.